Unable to compile Octokit-Monotouch in Xamarin Studio in mac

I was trying to compile Octokit-XamarinStudio in mac and I ran into this issue.

```
Compilation failed: 4 error(s), 0 warnings

error CS2001: Source file `/Users/naveen/code/octokit.net/Octokit/Models/Request/TestRepositoryHook.cs' could not be found
error CS2001: Source file `/Users/naveen/code/octokit.net/Octokit/Models/Response/RepositoryHookConfiguration.cs' could not be found
error CS2001: Source file `/Users/naveen/code/octokit.net/Octokit/Models/Request/RepositoryHooksPingRequest.cs' could not be found
error CS2001: Source file `/Users/naveen/code/octokit.net/Octokit/Models/Request/RepositoryHookTestRequest.cs' could not be found


Build complete -- 4 errors, 0 warnings

---------------------- Done ----------------------

Build: 4 errors, 0 warnings

```
And when I looked at the repo these files were missing.  

Thanks

--- a/Octokit/Octokit-Monotouch.csproj
+++ b/Octokit/Octokit-Monotouch.csproj
@@ -364,7 +364,6 @@
     <Compile Include="Models\Request\EditRepositoryHook.cs" />
     <Compile Include="Models\Request\NewRepositoryFork.cs" />
     <Compile Include="Models\Request\NewRepositoryHook.cs" />
-    <Compile Include="Models\Request\TestRepositoryHook.cs" />
     <Compile Include="Models\Response\RepositoryHook.cs" />
     <Compile Include="Models\Response\RepositoryPermissions.cs" />
     <Compile Include="Models\Response\CombinedCommitStatus.cs" />
@@ -393,16 +392,13 @@
     <Compile Include="Models\Response\AccountType.cs" />
     <Compile Include="Models\Request\ReleaseAssetUpload.cs" />
     <Compile Include="Models\Request\RepositoryRequest.cs" />
-    <Compile Include="Models\Response\RepositoryHookConfiguration.cs" />
     <Compile Include="Models\Request\RepositoryForksListRequest.cs" />
     <Compile Include="Models\Response\GitIgnoreTemplate.cs" />
     <Compile Include="Models\Response\License.cs" />
     <Compile Include="Models\Response\LicenseMetadata.cs" />
     <Compile Include="Models\Response\PullRequestFile.cs" />
     <Compile Include="Models\Request\PublicRepositoryRequest.cs" />
-    <Compile Include="Models\Request\RepositoryHooksPingRequest.cs" />
     <Compile Include="Exceptions\TwoFactorAuthorizationException.cs" />
-    <Compile Include="Models\Request\RepositoryHookTestRequest.cs" />
     <Compile Include="Http\HttpMessageHandlerFactory.cs" />
     <Compile Include="Models\Response\TeamMembership.cs" />
     <Compile Include="Models\Response\ResourceRateLimit.cs" />

